### PR TITLE
Internal Workshops

### PR DESCRIPTION
**Pull request by balanza**
Some rules and advices on internal workshop we periodically run. I created a brand new `learning.md` file as I thought it can include the broader argument of _care your knowledge_.

### CODE REVIEW

File: learning.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+We periodically run short internal workshop sessions to facilitate knowledge sharing between between us.
+Subject may vary: they can be about explaining some new domain concepts in IO as well as share with other the use of a specific technology or introducing new features in our common tools.
+
+In few words: anything you think your colleagues can benefit from.
+ 
+
+#### Instructions
+###### before
+* Take the initiative: propose yourself to run a workshop. Alternatively, ask a colleague of yours to do that if you think she/he knows something worth sharing. 
+* Once scheduled, create a public calendar event and share it.
```
Comment (gunzip) [inline learning.md]: - post the link to the event on #team_tech channel
- save all assets and sources (excalidraw, gdoc, slides, ...) and share them aside the video

File: learning.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+## Learning and knowledge
```
Comment (gunzip) [inline learning.md]: I'd call this 'Internal workshops'

File: learning.md (hunk 1 above)
Comment (balanza) [reply learning.md]: > post the link to the event on #team_tech channel

ok


> save all assets and sources (excalidraw, gdoc, slides, ...) and share them aside the video

it's said in the _after_ section

File: learning.md (hunk 2 above)
Comment (balanza) [reply learning.md]: so I rename the file as well?

Comment (gunzip) [reply learning.md]: yup

File: learning.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+### Resources
+* We have a [wiki](https://sites.google.com/pagopa.it/io-wiki/io-home).
+
+### Internal workshops
+We periodically run short internal workshop sessions to facilitate knowledge sharing between between us.
```
Comment (Undermaken) [inline learning.md]:
```suggestion
We periodically run short internal workshop sessions to facilitate knowledge sharing between us.
```

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -10,6 +10,7 @@ The idea is that the notebook will become something like [Gitlab’s](https://ab
 1. 📘 [How To Use This Handbook](how-to.md)
 1. ☎️ [Communication](communication.md)
 1. 👨‍💻 [Who Does What?](who-does-what.md)
+1. 🤯 [Learning and knowledge](learning.md)
```
Comment (cloudify) [inline README.md:13]:
```suggestion
1. 🤯 [Learning and Knowledge Sharing](learning.md)
```

File: learning.md (hunk 2 above)
Comment (cloudify) [reply learning.md]: I think `Learning and Knowledge Sharing` is better suited, as internal workshops are only a way to accomplish the goal of learning and knowledge sharing, we may add other kind of initiatives in the future

File: learning.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+Subject may vary: they can be about explaining some new domain concepts in IO as well as share with other the use of a specific technology or introducing new features in our common tools.
+
+In few words: anything you think your colleagues can benefit from.
+ 
+
+#### Instructions
```
Comment (cloudify) [inline learning.md]: we can remove this headline, doesn't add much value

File: learning.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+###### before
```
Comment (cloudify) [inline learning.md]:
```suggestion
#### How to organize a workshop
```

File: learning.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+* Take the initiative: propose yourself to run a workshop. Alternatively, ask a colleague of yours to do that if you think she/he knows something worth sharing. 
+* Once scheduled, create a public calendar event and share it. Post the link to the event on `#team_tech` channel
+* Define in advance an agenda with what you are going to share. You can use [Fellow](https://pagopa.fellow.app) for that.
+* Come prepared: take the right time to prepare slide/code/sandboxes in advance. Consider assign yourself a story on the issue tracker _(example: Plan typescript workshop)_.
+###### during
```
Comment (cloudify) [inline learning.md]:
```suggestion
#### How to run a workshop
```

File: learning.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+* Record the presentation so anyone can enjoy it later. Please refer to the [usual guideline for video recording](https://github.com/pagopa/io-handbook/blob/master/communication.md#video-recordings).
+###### after
```
Comment (cloudify) [inline learning.md]:
```suggestion
#### How to share the content of a workshop
```

File: learning.md (hunk 5 above)
Comment (balanza) [reply learning.md:13]: you mean `#### Instructions`?

**Review by gunzip — APPROVED** [review]

**Pull request merged**